AdminWebsite/AdminWebsite/ClientApp/src/app/services/bookings-list.service.spec.ts

```diff
@@ -270,12 +270,12 @@ export class ResponseTestData {
         return bookingsResponse;
     }
     getTestData(): BookingsByDateResponse {
-        const bhr = new BookingsHearingResponse();
+        const date = new Date('2019-10-22 13:58:40.3730067');
+        const bhr = new BookingsHearingResponse({ hearing_date: date });
         bhr.hearing_id = '1';
-        bhr.created_date = new Date('2019-10-22 13:58:40.3730067');
-        bhr.hearing_date = new Date('2019-10-22 13:58:40.3730067');
-        bhr.last_edit_date = new Date('2019-10-22 13:58:40.3730067');
-        bhr.scheduled_date_time = new Date('2019-10-22 13:58:40.3730067');
+        bhr.created_date = date;
+        bhr.last_edit_date = date;
+        bhr.scheduled_date_time = date;
```

Comment:
Not sure it matters but perhaps: 
`bhr.created_date = new Date(date)`

So that they aren't all the same reference.

Reply:
I don't think it does. The test is setup with a current point in time so I figured it would be cleaner to manage it with a single variable